osvr/RenderKit/RenderManager.h

```diff
@@ -612,8 +612,11 @@ namespace renderkit {
                 m_distortionParameters; ///< One set per eye x display
 
             bool m_enableTimeWarp;       ///< Use time warp?
+            bool m_justInTimeWarp;       ///< Use just-in-timewarp?
+                                         ///(requires enable)
+            float m_justInTimeWarpRotation; ///< Rotation in degrees, 0 = top, 90 = right
```

Comment:
This might be a can of worms, and applies to other fields like this, but why do we store m_enableTimeWarp and m_justInTimeWarp separately here when we have all the config information from the config object as parsed from OSVR-Core?